ios/FluentUI/IndeterminateProgressBar/IndeterminateProgressBar.swift

```diff
@@ -57,10 +57,13 @@ public struct IndeterminateProgressBar: View, ConfigurableTokenizedControl {
             .modifyIf(!state.isAnimating && state.hidesWhenStopped, { view in
                 view.hidden()
             })
-            .resolveTokens(self)
     }
 
-    var tokens: IndeterminateProgressBarTokens { state.tokens }
+    let defaultTokens: IndeterminateProgressBarTokens = .init()
+    var tokens: IndeterminateProgressBarTokens {
+        let tokens = tokens(for: fluentTheme)
```

Comment:
nit: why not just `return tokens(for: fluentTheme)` ?

Reply:
Ah no particular reason. Partly bulk changing of the property, partly some debugging I was doing. But I agree that that's cleaner, so I'll do that in all the cases where no props are changed.